### ISSUE TITLE
How do you secure authentication credentials ?

### ISSUE THREAD

**Issue by jalik** [sebakerckhof/ufs-wabs#1]
Just to know, how do you handle private credentials ?
I mean since you give account name and password in the Store initialization as parameters, they should not be visible on the client but only on the server for security reasons, so how do you handle this ? using a `Meteor.isServer` ?

**Comment by sebakerckhof** [comment]
No. Meteor isserver is not secure. You should use environment variables. But I ported this from my cfs storage adapter, where client and server is separated.  i forgot about this... I'll make some changes tomorrow

**Comment by jalik** [comment]
Okay, don't forget to mention it in the README (it's not obvious for some people unfortunately), so everyone will do it the right way ;)
Thank you.
